wifi_logger.c

```diff
@@ -192,11 +192,38 @@ void start_wifi_logger(void)
     #ifdef CONFIG_ROUTE_ESP_IDF_API_LOGS_TO_WIFI
     esp_log_set_vprintf(system_log_message_route);
     #endif
-    
+
     xTaskCreatePinnedToCore(&wifi_logger, "wifi_logger", 4096, NULL, 2, NULL, 1);
     ESP_LOGI(tag_wifi_logger, "WiFi logger initialised");
 }
 
+/*
+ * @brief A common wrapper function to check connection status for all interfaces
+ *
+ * @param handle_t Interface handle
+ * @return bool True if connected
+ */
+bool is_connected(void* handle_t)
```

Comment:
In case of UDP, this will return false. But it should return true, Add macro for udp too

something like
```c
#ifdef UDP
ret = true;
#endif
```

Reply:
not using it in UDP.

Reply:
Just for completeness

Reply:
done